pkgs/top-level/all-packages.nix

```
@@ -2217,6 +2217,8 @@ with pkgs;
 
   delta = darwin.apple_sdk_11_0.callPackage ../applications/version-management/delta { };
 
+  debase = darwin.apple_sdk_11_0.callPackage ../applications/version-management/debase { };
```

```suggestion
  debase = callPackage ../by-name/de/debase { inherit (darwin.apple_sdk_11_0) stdenv; };
```

This will allow putting in `by-name`